app/src/main/java/app/nexd/android/ui/helper/call/CallTranslateViewModel.kt

```diff
@@ -16,17 +15,14 @@ import app.nexd.android.api.model.HelpRequestCreateDto
 import io.reactivex.BackpressureStrategy
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.Disposable
-import io.reactivex.schedulers.Schedulers.io
 import okhttp3.ResponseBody
 import java.io.*
 import java.util.concurrent.TimeUnit
-import java.util.logging.Handler
-import kotlin.concurrent.fixedRateTimer
 
 class CallTranslateViewModel(application: Application) : AndroidViewModel(application) {
 
     companion object {
-        const val TMP_FILENAME = "callfile.wav"
+        const val TMP_FILENAME = "callTmpFile.wav"
```

Comment:
i think we should use cacheDir for this file so the system can clean it up if there is no storage left